tests/unit_tests/test_lib.py

```diff
@@ -339,6 +339,20 @@ def test_new_tally(lib_init):
     new_tally_with_id.scores = ['flux']
     assert len(openmc.lib.tallies) == 5
 
+def test_add_and_delete_tally(lib_init):
+    with pytest.raises(exc.AllocationError):
+        openmc.lib.Material(1)
+    new_tally = openmc.lib.Tally()
+    new_tally.scores = ['flux']
+    new_tally_with_id = openmc.lib.Tally(10)
+    new_tally_with_id.scores = ['flux']
+    assert len(openmc.lib.tallies) == 5
+    # delete tally and check length is one less than before
+    openmc_remove_tally(10)
+    assert len(openmc.lib.tallies) == 4
+
+# def test_delete_wrong_id():
```

Comment:
@pshriwise I think i should add a test to delete a wrong ID here, but I'm not sure exactly what needs to go here. Not sure if I should just use the `lib_init` and then try and delete something like `delete(9876243184073)` or if I should do something from scratch so I'm exactly sure which tally IDs exist in my setup. Do you have thoughts?

Reply:
I'd recommend gathering all of the tally ID's from the `lib_init` model and then 
try to delete the `max(IDs) + 1`.

Reply:
That is a good idea! I try that out